systemd/user/kano-common-refresh-token.timer

```diff
@@ -0,0 +1,16 @@
+#
+# kano-common-refresh-token.timer
+#
+# Systemd timer service to periodically refresh the Kano World login token.
+#
+# This timer is bound to the kano-common.target file,
+# and it will call the Unit file described below when triggered.
+#
+# Use "systemctl --user list-timers --all" to query the status of this timer.
+#
+
+[Timer]
+OnBootSec=60sec
+OnUnitActiveSec=60min
```

Comment:
If this timer does not use something more advanced (for example, "run `kano-common-refresh-token.service` every week on Tue") and simply running a unit on boot after x sec, then I recommend ditching this altogether and moving the "sleep" inside the python script itself. In my brief play with timers, the `OnBootSec` sometimes worked sometimes didn't which made it horrible to debug...

Reply:
I didn't experience these troubles, but if we don't like it I can move the sleep out of the timer. There's a pretty clear explanation on this parameter, here:

 * https://github.com/systemd/systemd/issues/4572#issuecomment-260014156

Also, `systemctl --user list-timers` provides a very accurate diagnosis on these timers, which is pretty useful for debugging them. I actually used it to document the QA specs.